### PR TITLE
Proper cleanup for vpa.

### PR DESCRIPTION
**Pull request by bskiba**
Add information on how to tear down vpa.
Add script deleting webhook on teardown.

### CODE REVIEW

**Comment by bskiba** [comment]
@schylek

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -151,6 +151,27 @@ or recreated by their controller due to Updater's activity).
...
 
+### Tear down
+
+Note that if you stop running VPA in your cluster, the resource requests
+for the pods already modified by VPA will not change, but any new pods
+will get resources as defined in your controllers (i.e. deployment or
+replicaset) and not according to previous recommendations made by VPA.
+
+To stop using Vertical Pod Autoscaling in your cluster:
+* Remove any verticalpodautoscaler objects you created. For example:
```
Comment (kgrygiel) [inline vertical-pod-autoscaler/README.md]: I think this step shouldn't be necessary if you do ./hack/vpa-down.sh, which deletes the CRD and so removes all VPA objects.

File: vertical-pod-autoscaler/pkg/admission-controller/delete-webhook.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Unregisters the admission controller webhook.
+set -e
+
+echo "Unregistering VPA asmission controller webhook"
```
Comment (kgrygiel) [inline vertical-pod-autoscaler/pkg/admission-controller/delete-webhook.sh]: "admission"

File: vertical-pod-autoscaler/README.md (hunk 1 above)
Comment (bskiba) [reply vertical-pod-autoscaler/README.md]: Done

**Pull request merged**